lib/Makefile.am

```diff
@@ -71,10 +71,6 @@ check:
 test:
 	$(CARGO) test $(CARGO_MANIFEST_ARG) $(CARGO_EXTRA_ARGS)
 
-.PHONY: clippy
-clippy:
-	$(CARGO) clippy $(CARGO_MANIFEST_ARG) $(CARGO_EXTRA_ARGS) -- -Dwarnings
```

Comment:
hmm.. why remove this line?? we can do `#[allow(clippy::<lint>)]` specifically...